adapters/openx/openx_test.go

```diff
@@ -33,6 +34,104 @@ func TestResponseWithCurrencies(t *testing.T) {
 	assertCurrencyInBidResponse(t, "EUR", &currency)
 }
 
+func TestGetBidMeta(t *testing.T) {
```

Comment:
Let's name it similar as the test below:
```suggestion
func TestOpenxAdapter_GetBidMeta(t *testing.T) {
```